zerver/tests/test_outgoing_webhook_system.py

```diff
@@ -29,43 +31,54 @@ def timeout_error(http_method, final_url, data, **request_kwargs):
     # type: (Any, Any, Any, Any) -> Any
     raise requests.exceptions.Timeout
 
+class MockServiceHandler(object):
```

Comment:
Nit: Do you think this be `MockServiceHandler(OutgoingWebhookServiceInterface)` instead?  Also, would it be helpful to include the `process_command` method as well?